src/algorithm.jl

```diff
@@ -17,74 +17,72 @@ function optimize!(sp::AbstractStochasticProgram, algo::AbstractAlgorithm, stopc
     # Default implementation, define a specific method for algorithms for which
     # this default is not appropriate
     mastersol = nothing
-    totalstats = SDDPStats()
-    stats = SDDPStats()
+    totalstats = Stats()
```

Comment:
Concerning stats, I think we should allow the user to store the trace along iterations. 
For instance, concatenate all `stats` in a `Vector{Stats}`: 

```
push!(trace, stats)

```